### DISCUSSION TITLE
Configurable api domain and docs path

### DISCUSSION THREAD

**Discussion by m-lotze** [dedoc/scramble#45] [Ideas]
We have one application where we use a subdomain for the api. It would be good to have an option to configure the domain for the api and maybe an option for the docs path.

```

'api_domain' => env('API_DOMAIN'),
'api_path' => '',
'docs_path => 'docs'

```

If the domain is set, the routes could be filtered by default:

```

Scramble::routes(fn (Route $route) => $route->getDomain() === config('scramble.api_domain'));

```

**Comment by romalytvynenko** [comment]
Hey @m-lotze 

Will add API domain to config.

Can you please elaborate on why you need other route (customizable `docs_path`)?

**Reply by m-lotze** [reply]
It depends what the docs path would be, if domain is set to `api.domain.com` and `api_path` to an empty string. If it is `api.domain.com/docs` than there is no need to change it.

**Comment by romalytvynenko** [comment]
Closing as made that possible: https://scramble.dedoc.co/usage/info
